sdk/search/Azure.Search.Documents/src/SearchFilter.cs

```diff
@@ -74,15 +72,13 @@ public static string Create(FormattableString filter, IFormatProvider formatProv
                     DateTimeOffset x => JsonSerialization.Date(x, formatProvider),
                     DateTime x => JsonSerialization.Date(x, formatProvider),
 
-#if EXPERIMENTAL_SPATIAL
                     // Points
                     GeoPosition x => EncodeGeography(x),
                     GeoPoint x => EncodeGeography(x),
 
                     // Polygons
-                    GeoLine x => EncodeGeography(x),
+                    GeoLineString x => EncodeGeography(x),
```

Comment:
Did we want this to be a closed line or any line? If we're looking for a guaranteed closed line this should be `GeoLinearRing`.

Reply:
In general, there are no restrictions on the nature of the coordinates in a `GeoLineString`.
`EncodeGeography(GeoLineString)` enforces that the coordinates represent a closed ring.